pkg/client/cache/reflector.go

```diff
@@ -265,6 +265,7 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 			switch err {
 			case io.EOF:
 				// watch closed normally
+				glog.V(1).Infof("%s: Watch for %v closed normally (with EOF)", r.name, r.expectedType)
```

Comment:
OK will take it down to V(4).  Worried that it won't appear in our integration test flakes, but hopefully we run them at >= V(4)...

I opened #20659 to track whether we should restart the watch here, rather than burying it in this PR.